.travis.yml

```diff
@@ -0,0 +1,53 @@
+language: python
+python:
+  - "3.7"
+# command to install dependencies
+before_install:
+  - pip install -U pip 
+  
+install:
+  - pip install -r requirements.txt
+  - pip uninstall -y overrides
+  - pip install overrides==3.1.0
```

Comment:
Why not just set the right version in requirements.txt? This seems to make installation harder for users.